android/quest/src/main/java/org/smartregister/fhircore/quest/ui/report/measure/MeasureReportViewModel.kt

```diff
@@ -190,15 +198,21 @@ constructor(
           }
         }
       }
-      is MeasureReportEvent.OnPatientSelected ->
+      is MeasureReportEvent.OnPatientSelected -> // Reset previously selected patient
+        //  Update dateRange and format start/end dates e.g 16 Nov, 2020 - 29 Oct, 2021
```

Comment:
Add a `TODO` for this comment for future implementation.

Reply:
@ellykits  I think this is not TODO for future, This is for explaining what this chunk of code will do.